crates/stake-chain/Cargo.toml

```
@@ -0,0 +1,35 @@
+[package]
+edition = "2021"
+name = "strata-bridge-stake-chain"
+version = "0.1.0"
+
+[lints]
+rust.missing_debug_implementations = "warn"
+rust.rust_2018_idioms = { level = "deny", priority = -1 }
+rust.unreachable_pub = "warn"
+rust.unused_crate_dependencies = "deny"
+rust.unused_must_use = "deny"
```

I did not add the `docs` lints in the `tx-graph` crate because it was mostly carried over from the PoC. For this crate, since we're starting from scratch, we should add the docs lints:

```suggestion
rust.unused_must_use = "deny"
rust.missing_docs = "warn"
rustdoc.all = "warn"
```